client/thin-replica-client/include/client/thin-replica-client/trace_contexts.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2021 VMware, Inc. All rights reserved. VMware Confidential
+
```

Comment:
Does this file not need one of those license statements that begins "This product is licensed to you under..."?